.github/workflows/build-monorepo.yml

```diff
@@ -0,0 +1,76 @@
+name: Test build in monorepo
+on:
+  workflow_call:
+    inputs:
+      runs_on:
+        required: true
+        type: string
+      os:
+        required: true
+        type: string
+      concurrency_group:
+        required: true
+        type: string
+      is_hoisted:
+        required: true
+        type: boolean
+
+jobs:
+  build:
+    name: Build monorepo with Reanimated
+    runs-on: ${{ inputs.runs_on }}
+    concurrency:
+      group: ${{ inputs.concurrency_group }}
+      cancel-in-progress: true
+    steps:
+      - name: Create monorepo
+        run: |
+          mkdir monorepo
+          cd monorepo
+          echo '{"name":"rnos-monorepo-tester","version":"1.0.0","license":"MIT","private":true,"workspaces":{"packages":["RootApp","packages/PackageApp", "AppA", "AppB"],"nohoist":["**/react","**/react-dom","**/react-native","**/react-native/**","**/react-native-codegen","**/react-native-dev-menu"]}}' > package.json
+          yarn
+          mkdir packages
+      - name: Create RootApp
+        working-directory: monorepo
+        run: npx react-native@next init RootApp
+      - name: Create PackageApp
+        working-directory: monorepo/packages
+        run: npx react-native@next init PackageApp
+      
+      - name: Install dependencies for RootApp
+        working-directory: monorepo/RootApp
+        run: yarn add github:software-mansion/react-native-reanimated#${{ github.ref }}
```

Comment:
Is it possible to install app from `file:../react-native-reanimated` instead?

Also, for v2 this checks monorepo support only with building from source enabled. Maybe we should have another job that first builds the package with `createNPMPackage.sh`, then installs Reanimated from *.tgz file and finally tries to build the app?

Reply:
We need to create another PR for this feature